tabular/tests/unittests/test_tabular.py

```diff
@@ -694,7 +692,7 @@ def test_pseudolabeling():
                     assert False, error_msg + "unlabeled test data, best quality" + error_suffix
 
 
-@pytest.mark.slow
+@pytest.mark.platform
```

Comment:
We probably want to mark this test both slow and platform. Otherwise, this would change behavior in our regular CI